server/trucks.js

```diff
@@ -248,9 +251,23 @@ var foodTrucks = [
 	}
 ];
 
-/* 
- * return an object from the module with a single method on it: filterByDay
- * that method should take a single parameter that represents the day to filter on
- * use underscore's filter() method to return all trucks that have the day in their
- * schedule
- * /
\ No newline at end of file
+
+var filterTrucks = {
+
+	// note: only using underscore to check list contents,
+	// could not figure out easy way to use filter method in this instance
+
```

Comment:
Underscore's filter method takes two arguments: a list, and a predicate function that returns true or false - if it returns true, the item being checked will be returned in the list that filter returns. So something like this:
`function filterByDay(requestedDay) {
  return _.filter(foodTrucks, function (truck) { return _.contains(truck.schedule, requestedDay); });
}`